### PR TITLE
Initial Text for the Mesos GSG

### PR DESCRIPTION
**Pull request by techcet**

### CODE REVIEW

File: Documentation/gettingstarted.rst [hunk 1]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+
+If you don't already have Vagrant installed, follow the
+`Vagrant Install Instructions <https://www.vagrantup.com/docs/installation/>`_
+or see `Download Vagrant <https://www.vagrantup.com/downloads.html>`_ for newer versions.
+
+
+Step 1: Download the Cilium Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Download the latest Cilium `source code <https://github.com/cilium/cilium/archive/master.zip>`_
```
Comment (tgraf) [inline Documentation/gettingstarted.rst:885]: We should fix all guides to not point to the master release as it is a nightly generated tarball of the latest development snapshot. I'll create a separate issue to track this. Unfortunately, GitHub does not make it simple to link to the latest stable release so we'll have to maintain a stable URL ourselves.

File: Documentation/gettingstarted.rst [hunk 2]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+    ContainerRuntime:   Ok
+    <...>
+    Cilium:             Ok
+
+The status indicates that all necessary components are operational.
+
+Step 5: Run Script to Start Marathon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start Marathon inside of the vbox:
```
Comment (tgraf) [inline Documentation/gettingstarted.rst]: The reset of the guides refers to *Vagrant VM* instead of box

File: Documentation/gettingstarted.rst [hunk 3]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+
+::
+
+    $ ./start_marathon.sh
+
+Step 6: Start a Simple Web-Server and Client 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The web-server simply responds to requests on a particular port. Use the ``curl`` command below to launch it.
```
Comment (tgraf) [inline Documentation/gettingstarted.rst]: This needs a short explanation that curl is used to talk to the Marathon API to schedule a task

File: Documentation/gettingstarted.rst [hunk 4]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+
+::
+
+    $ curl -i -H 'Content-Type: application/json' -d @web-server.json 127.0.0.1:8080/v2/apps
+
```
Comment (tgraf) [inline Documentation/gettingstarted.rst:976]: I would add a note that the output should be:
```
HTTP/1.1 201 Created
[...]
```
As the extensive response from Marathon looks a bit like something went wrong.

File: Documentation/gettingstarted.rst [hunk 5]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+Confirm that Cilium sees the new workload. The output should return the endpoint with label ``mesos:id=web-server`` and the assigned IP:
+
+::    
+
+    $ cilium endpoint list
```
Comment (tgraf) [inline Documentation/gettingstarted.rst:991]: People also read through guides without running them. Listing the output of commands is thus helpful:
```
ENDPOINT   POLICY        IDENTITY   LABELS (source:key[=value])   IPv6                   IPv4           STATUS
           ENFORCEMENT
29898      Disabled      256        mesos:id=web-server           f00d::a00:20f:0:74ca   10.15.242.54   ready
```

File: Documentation/gettingstarted.rst [hunk 6]
```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
...
+
+::
+
+    $ cilium endpoint list
+    ENDPOINT   POLICY        IDENTITY   LABELS (source:key[=value])   IPv6                   IPv4           STATUS   
+               ENFORCEMENT                                                                                           
+    29898      Disabled      256        mesos:id=web-server           f00d::a00:20f:0:74ca   10.15.242.54   ready    
+    33115      Disabled      257        mesos:id=client               f00d::a00:20f:0:815b   10.15.220.6    ready
+
+There is no easy way to retrieve the logs of the client, so use the following script:
```
Comment (tgraf) [inline Documentation/gettingstarted.rst]: I would rephrase this so it doesn't sound like Cilium is at fault for not providing easier log access. I would write something like this:

Marathon runs all tasks as batch jobs and makes the stdout output available via a log file in `/var/lib/mesos`. The `tail_client.sh` script will identify the log file and then run `tail` on it.

**Review by tgraf — CHANGES_REQUESTED** [review]
Looks great, a couple of comments inline

**Review by ianvernon — APPROVED** [review]

File: Documentation/gettingstarted.rst (hunk 5 above)
Comment (techcet) [reply Documentation/gettingstarted.rst:991]: Agreed- done.

File: Documentation/gettingstarted.rst (hunk 4 above)
Comment (techcet) [reply Documentation/gettingstarted.rst:976]: Agreed- done.

File: Documentation/gettingstarted.rst (hunk 2 above)
Comment (techcet) [reply Documentation/gettingstarted.rst]: Done.

**Review by tgraf — APPROVED** [review]

**Review by amreshakim — APPROVED** [review]
Looks awesome! Thanks for doing this.
